### DISCUSSION TITLE
Donde entregamos la practica de la semana 5

### DISCUSSION THREAD

**Discussion by MiguelAngelAguilar312** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#548] [Q&A]
Buenas noches cual es el link para entregar los repositorios de la semana 5

**Comment by doguedogue** [comment]
creo que no mandarón el link

**Comment by JuanSalvador73** [comment]
No han publicado el link. Como la dinámica de paso a BackEnd ha cambiado según el aviso dado por LaunchX, parece que al final solo nos pedirán a los de FrontEnd dar nuestros enlaces de perfil con repositorios en algún formulario general que nos enviarán. Eso ya lo sabremos este jueves 31 de marzo.
